android/filament-android/src/main/java/com/google/android/filament/Scene.java

```diff
@@ -18,6 +18,26 @@
 
 import android.support.annotation.Nullable;
 
+/**
+ * A <code>Scene</code> is a flat container of {@link RenderableManager} and {@link LightManager}
+ * components.
+ * <br>
+ * <p>A <code>Scene</code> doesn't provide a hierarchy of objects, i.e.: it's not a scene-graph.
+ * However, it manages the list of objects to render and the list of lights, these can
+ * be added or removed from a <code>Scene</code> at any time.</p>
```

Comment:
I would add: "Moreover clients can use {@link TransformManager} to create a graph of transforms."